src/components/EntryListItem.vue

```diff
@@ -1,13 +1,21 @@
 <template>
 	<div class="entry-list-item selectable between flair" v-bind:class="{ active: entry.view_is_active }" v-on:click="details">
 		<div class="text-info" v-bind:class="{ strike: entry.is_expired }">
-			<span class="header">{{ header }}</span>
-			<br>
-			<span class="user">
-        {{ entry.userName || '&#60;empty&#62;' }}
-      </span>
+			<div class="header">
+				{{ header }}
+			</div>
+			<div class="user">
+				{{ entry.userName || '&#60;empty&#62;' }}
+			</div>
+		</div>
+		<div class="otp-block valign-wrapper" v-bind:class="{ hidden: !entry.view_is_active }">
```

Comment:
So this will only show when hotkey navigation is enabled and the current entry is active?